docs/UserGuide.md

```diff
@@ -154,7 +154,7 @@ Example: `list` shows all the employees in the company
 Removes all the employees’ information in the company from the app. Useful for clearing out sample data.<br>
 ❗️Caution: You cannot recover the data afterwards.
 
-[coming in v1.3] Requests for confirmation from the user before clearing. // ian's idea: get them to type 'yes clear' to confirm
+[coming in v1.3] Require confirmation with a `yes clear` for clearing all data with `clear`. // ian's idea: get them to type 'yes clear' to confirm
```

Comment:
- technically 1.3 is over, maybe we can remove it entirely or say it's coming in some super-far-off future version
- should probably remove the comment too